reacnetgenerator/utils.py

```
@@ -273,6 +276,264 @@ def bytestolist(x: bytes) -> Any:
+    except StopIteration:
+        pass
+    else:
+        raise RuntimeError("Ordered result count does not match declared total")
+    if pending or spool.pending_count:
+        raise RuntimeError("Ordered result count does not match declared total")
+
+
+def _bounded_multiopen(
+    pool,
+    func: Callable,
+    l: IO,
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)